src/Formatter/FormatterInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Greg0ire\EnumBundle\Formatter;
+
+
+interface FormatterInterface
+{
+    /**
+     * @param $text
+     * @return string
+     */
+    public function transform($text);
```

Comment:
Could you also rename this to `format` ? Transformers transform, formatters format…